### PR TITLE
fix(profiles) - add profile to client initialization

### PR DESCRIPTION
**Pull request by giladlevy7**
Add profile name to the client initialization

### CODE REVIEW

File: rivery_cli/cli/activities.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def download_run_logs(ctx, **kwargs):
     run_id = kwargs.get('runid')
 
     click.echo(f'Starting to download logs for run id {run_id} in profile: {profile_name}')
-    rivery_client = client.Client(name=profile_name)
+    rivery_client = client.Client(name=profile_name, profile=profile_name)
```
Comment (Alonreznik) [inline rivery_cli/cli/activities.py:79]: This should come from the client from the context :)

File: rivery_cli/cli/rivers.py [hunk 2]
```diff
@@ -400,7 +400,7 @@ def wait_for_end(session, run_id, timeout=3600 * 2):
 def check_run(ctx, **kwargs):
     """ Check the run status by runId """
     profile_name = ctx.get('PROFILE')
-    rivery_client = client.Client(name=profile_name)
+    rivery_client = client.Client(name=profile_name, profile=profile_name)
```
Comment (Alonreznik) [inline rivery_cli/cli/rivers.py:403]: Same as above:
This should come from the client from the context :)

**Review by Alonreznik — CHANGES_REQUESTED** [review]

File: rivery_cli/cli/activities.py (hunk 1 above)
Comment (giladlevy7) [reply rivery_cli/cli/activities.py:79]: Talked F2F

File: rivery_cli/cli/rivers.py (hunk 2 above)
Comment (giladlevy7) [reply rivery_cli/cli/rivers.py:403]: Talked F2F

**Review by Alonreznik — APPROVED** [review]

**Pull request merged**